components/playback/controls/player/InputWidget.vue

```diff
@@ -92,6 +94,8 @@ export default {
     input_width: { type: Number, default: 0 }, // textbox_width (int)  [pixels]
     disabled: { type: Boolean, default: false }, // disabled (optional bool=False) (not able to type into input)
     dom_id_suffix: { type: String, default: "" }, // TODO (Eli 11/3/20): consider defaulting this to a random UUID if no value supplied
+    display_text_message: { type: Boolean, default: true }, // display_text_message (boolean) if set to false would not render invalid_text
+    cut_paste_disable: { type: Boolean, default: false }, // cut_paste_disable (boolean) if set to true would prevent cut and paste of text into input
```

Comment:
this only disables pasting right? it doesn't disable the "cut" inside the text input field, correct?
If that's true, then I think it should only be named `disable_paste`.  There may be other situations in the future where for some reason we want to disable cut inside the text field, but that is not the current situation

Reply:
Modified the api and the prop name to disable_paste